scripts/export_package.sh

```diff
@@ -3,7 +3,8 @@ set -e
 
 . venv/bin/activate
 
-gitchangelog
+sudo gem install github_changelog_generator
```

Comment:
Does ruby come preinstalled in your circle docker image?

Reply:
Nope, this export_package.sh script is semi-automated ;), only currently run in master branch by admins